crates/tests-e2e/src/tracing.rs

```diff
@@ -0,0 +1,14 @@
+use std::sync::Once;
+
+pub fn init_tracing() {
+    static TRACING_TEST_SUBSCRIBER: Once = Once::new();
+
+    TRACING_TEST_SUBSCRIBER.call_once(|| {
+        tracing_subscriber::fmt()
+            .with_env_filter(
+                "debug,hyper=warn,reqwest=warn,rustls=warn,bdk=info,lightning=trace,sled=info,lightning::chain::channelmonitor=trace,lightning::ln::peer_handler=debug,ureq=info",
```

Comment:
🔧 Let's use multi-line formatting here so that it's easier to read diffs. We do the same in the `ln-dlc-node` tests now.

Reply:
I just happened to copy-paste before you fixed the other occurrence 🙃 
will align.